src/components/select/editForm/Select.edit.data.js

```diff
@@ -271,12 +271,30 @@ export default [
       }
     }
   },
+  {
+    type: 'textfield',
+    input: true,
+    key: 'sort',
+    label: 'Sort Query',
+    weight: 18,
+    description: 'The sort query for results',
+    tooltip: 'User this to provide additional sorting using query parameters',
+    conditional: {
+      json: {
+        or: [
```

Comment:
Please consider usage of 'in' operator here http://jsonlogic.com/operations.html#in

Reply:
This can be resolved in another PR. going to accept it as is.